### ISSUE TITLE
[1.16.5] Dynmap wont work

### ISSUE THREAD

**Issue by DNAmaster10** [Teneted/Tenet#1435]
**Minecraft Version :** 1.16.5

**Mohist Version :** mohist-1.16.5-5-server.jar

**Operating System :** Windows 10

**Concerned mod / plugin** : [Dynmap ](https://www.spigotmc.org/resources/dynmap.274/)

**Logs :** [here](https://drive.google.com/file/d/1SV5AAaDR3S7nYjHwFPK9IU6OsK8FyTtj/view?usp=sharing)

**Steps to Reproduce :**
Install Dynmap in plugins folder

**Description of issue :** I've tried using multiple versions of Mohist, I've tried using multiple versions of Dynmap. The error still seems to persist. Thanks if anyone can help!

**Comment by Shawiizz** [comment]
Paper builds are outdated, don't use them. Please use latest build (https://mohistmc.com/api/1.16.5/latest/download)

**Comment by DNAmaster10** [comment]
> 
> 
> Paper builds are outdated, don't use them. Please use latest build (https://mohistmc.com/api/1.16.5/latest/download)

Thank you i'll try that now and i'll let you know if that fixes the issue

**Comment by Shawiizz** [comment]
Another thing, please use latest build of dynmap, because the one you use isn't supported by 1.16.5.

**Comment by DNAmaster10** [comment]
Thank you, but as i mentioned the error persisted across multiple versions. Sorry if my English is slightly bad!

**Comment by DNAmaster10** [comment]
I just installed the latest version of Dynmap and the latest version of Mohist, and for whatever reason, i'm still getting the error. The new log is[ here](https://drive.google.com/file/d/1RQYS9vpPnV_aySjFGb9g9HfZlu6_3yMZ/view?usp=sharing)

**Comment by Shawiizz** [comment]
Okay, this seems to be a bug, i add the bug label

**Comment by DNAmaster10** [comment]
Thankyou!

**Comment by SebastianClarke0** [comment]
I am also having the same issue with my server.

**Comment by JamesBayhaner** [comment]
Did you find a fix for this, @DNAmaster10 ?

**Comment by DNAmaster10** [comment]
> 
> 
> Did you find a fix for this, @DNAmaster10 ?

Hi James. As @Shawiizz mentioned, this is a big with Mohist. Having zero experience with Java, I have yet to find any fix to this issue, although I've tried many things. After messing around with the plugin a bit, I found that the Forge version of Dynmap does work correctly if you wanted to use that, however this is unfortunately not suitable for me as my server relies heavily on the use of Dynmap Towny, a plugin which displays the borders of peoples towns on the map. Im sure the Mohist dev team are already onto the issue though, James, so best not to rush them.
